### PR TITLE
docs: profile basic information regarding external registry using pro…

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR adds the ability to configure external registries using cluster profiles instead of user data. 

## Changed Pages

💻 [Preview URL for Page](https://deploy-preview-3661--docs-spectrocloud.netlify.app/clusters/edge/site-deployment/deploy-custom-registries/local-registry/)

## Jira Tickets

🎫 [DOC-1340](https://spectrocloud.atlassian.net/browse/DOC-1340)

## Backports

Can this PR be backported?

- [ ] Yes. 

[DOC-1340]: https://spectrocloud.atlassian.net/browse/DOC-1340?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by rishi-anand** [comment]
@kirankn8 Please review.. Do we need `data.name` in secret?

**Review by rishi-anand — APPROVED** [review]
This looks good @lennessyy 👍

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md [hunk 1]
```diff
@@ -78,6 +80,89 @@ to <VersionedLink text="Harbor Edge-Native Config pack" url="/integrations/packs
...
+configured an external registry through user data, you no longer need to provide credentials to the private registries
+referenced in the profile because those private registries will not be used. You are instead responsible for ensuring
+that all images are present in the external registry. For more information, refer to
+[Deploy with External Private Registry](./deploy-external-registry.md).
+
+:::
+
+If your have not configured an external registry for your Edge host through the Edge installer configuration user data,
+and your cluster profile references private registries, you must provide credentials to those private registries through
+Kubernetes secrets using a manifest in your cluster profile.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md:101]:
```suggestion
[Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/) using a manifest in your cluster profile.
```
I think adding a link to the k8s secret ref page might help those that may not understand what a k8s secret is 😄

**Review by karl-cardenas-coding — APPROVED** [review]
Very detailed and helpful guide @lennessyy

**Pull request merged**